libcaf_io/caf/io/basp/instance.hpp

```diff
@@ -294,11 +289,11 @@ class instance {
   bool handle(execution_unit* ctx, const Handle& hdl, header& hdr,
               std::vector<char>* payload, bool tcp_based,
               optional<endpoint_context&> ep, optional<uint16_t> port) {
-    // function object for checking payload validity
+    // Function object for checking payload validity.
```

Comment:
It's good to see your attention to detail. However, all of these changes are ideally wrapped into a single commit. Mixing style changes with functional changes blows up this commit and also blurs the step-by-step evolution a Git history ideally should represent.